README.md

```diff
@@ -40,249 +12,127 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+# 📖 [Yummy Meal] <a name="about-project"></a>
 
-## 🛠 Built With <a name="built-with"></a>
+The Yummy meal is a web application that utilizes an external API from 
+[https://www.themealdb.com/api/json/v1/1/filter.php?a=British] to showcase a list of meals. Users have the ability to interact with the meals by liking them, leaving comments, or reserving a meal using an involvement API. 
 
-### Tech Stack <a name="tech-stack"></a>
+However, the reservation feature has not been incorporated into the application due to limited group members. This project is the second Microverse capstone project designed to summarize the knowledge acquired in module 2. 
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
+The objective of the project is to develop a dynamic web application that meets the standards of a professional web developer.
+## Project requirements 
+Please find project requirements [here](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md). 
 
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
-
-<!-- Features -->
-
-### Key Features <a name="key-features"></a>
-
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## Project presentation
+## Screenshot
+![screenshot](./assets/Annotation%202023-03-16%20203224.png)
 
-<!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo"></a>
+## Please take a look at the video below
+<!-- The zoom video -->
+[Video Recording](https://www.loom.com/share/553f94cbe7924a1f9a296eb9e0afc789)
+## How it works
 
-> Add a link to your deployed project.
+The Meal DB web application allows users to like, comment, and make a reservation for their favorite meal.
+* By clicking on `like button` the number of like is increased and stored in the involvement API;
+* By clicking on the `Comment button` to display the description of the selected meal, the list of all comments, and added a new comment;
+* By clicking on the `Reservation button` to display the list of all reservations and make a new Reservation;
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
+## Live Demo
 
-## 💻 Getting Started <a name="getting-started"></a>
+[Live demo here](https://Cabraham1.github.io/capstone-project/)
```

Comment:
As previously mentioned, the live link is still not leading to your project, instead I get a 404 page error. Kindly address this to improve the documentation.

Reply:
done

Reply:
## 👍